npm-packages/git-cz-config/CHANGELOG.md

```
@@ -0,0 +1,165 @@
+# [1.3.0](https://github.com/zhumeisongsong/multiple-products-workspace/compare/git-cz-config-v1.2.4...git-cz-config-v1.3.0) (2024-12-15)
+
+
+### Bug Fixes
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)